kobo/apps/kobo_auth/models.py

```diff
@@ -87,3 +90,37 @@ def sync_to_openrosa_db(self):
             unique_fields=['pk'],
         )
         update_autofield_sequence(User)
+
+
+class DataCollectorGroup(models.Model):
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, on_delete=models.SET_NULL
+    )
+    uid = KpiUidField(uid_prefix='dcg', primary_key=True)
+    name = fields.CharField(max_length=200)
+
+    def __str__(self):
+        return '{} ({})'.format(self.name, self.uid)
+
+
+class DataCollector(models.Model):
+    uid = KpiUidField(uid_prefix='dc', primary_key=True)
+    name = fields.CharField(null=True, blank=True, max_length=200)
+    token = fields.CharField(max_length=40)
```

Comment:
I would leave a little more room enough if our token is smaller. In the future, if we want to increase the token length,  we won't have to create another migration.  
What about 64 characters?